src/components/02-components/branding-bar/branding-bar.stories.tsx

```diff
@@ -11,19 +11,16 @@ export default meta;
 
 export const Default: StoryObj<typeof BrandingBar> = {
   args: {
-    homeLinks: [
-      { title: 'Personal Information', url: '#' },
-      { title: 'Forms', url: '#' },
-      { title: 'Tasks', url: '#' },
-      { title: 'Docs', url: '#' },
-    ],
     siteLinks: [
+      { title: 'Staff', url: '#' },
       { title: 'Student', url: '#' },
       { title: 'Faculty', url: '#', isActive: true },
       { title: 'Partner', url: '#' },
     ],
-    helpLink: '#',
     myAccountLink: '#',
     logoutLink: '#',
+    dialogTitle: 'Students',
+    dialogText:
+      '<p>For assistance please contact the campus program office(s) with which you are working. If you are having technical issues, please contact the <a href="mailto:s4help@calstate.edu ">Help Desk</a></p>',
```

Comment:
@mcortes19 I don't know if it matters but there is an extra space at the end of the email address for the mailto link and when I click the mailto link it doesn't actually do anything ...

Reply:
@ccjjmartin great catch, I did  push the fix

Reply:
@mcortes19 🚢 The mailto link does work, I just found it on a different screen